addons/skin.estuary/xml/DialogPVRChannelManager.xml

```diff
@@ -30,18 +30,18 @@
 					<left>0</left>
 					<top>10</top>
 					<width>672</width>
-					<height>750</height>
+					<height>810</height>
 					<texture border="40">buttons/dialogbutton-nofo.png</texture>
 				</control>
 				<control type="list" id="20">
 					<left>20</left>
 					<top>30</top>
 					<width>630</width>
-					<height>710</height>
+					<height>770</height>
 					<onup>20</onup>
 					<ondown>20</ondown>
 					<onleft>9000</onleft>
-					<onright>60</onright>
+					<onright>9002</onright>
```

Comment:
it's now not possible to focus the scrollbar anymore, so please revert this change